clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/VCVTrait.scala

```diff
@@ -101,22 +97,7 @@ object VCVTrait {
       name = preferredName,
       alternateNames = alternateNames.toArray,
       `type` = rawTrait.extract("@Type").map(_.str),
-      xrefs = finalXrefs.toArray
+      xrefs = finalXrefs.toArray.distinct
```

Comment:
If we use a `Set` instead of a `List` for the accumulator, the refs should be de-duped as we build up the collection.